src/core/evaluation/expressionHandler.ts

```diff
@@ -22,38 +23,44 @@ const math = create({
   divideDependencies,
   SymbolNodeDependencies,
   FunctionNodeDependencies,
+  sumDependencies,
 });
 
 export default class ExpressionHandler {
   sheet: Sheet;
 
-  private cellRefCache: Array<PositionInfo>;
+  private cellRefHolder: Array<PositionInfo>;
+  private value: string;
```

Comment:
This name could be more specific. Maybe `rawValue`, following the pattern in #99, or `cellRawValue`, `cellFormula` etc.